src/test/java/com/exactpro/th2/FixHandlerTest.java

```diff
@@ -341,6 +365,7 @@ void updateTagTest() {
 class Channel implements IChannel {
     private final FixHandlerSettings fixHandlerSettings;
     private final MyFixHandler fixHandler;
+    private final FixHandler originalFixHandler;
```

Comment:
Why it is here? As I see it used only in 2 methods - it can moved out of `Channel` or even created in-place in these 2 methods